src/util/std_types.h

```diff
@@ -889,6 +889,16 @@ class code_typet:public typet
     return set(ID_access, access);
   }
 
+  bool get_is_constructor() const
+  {
+    return get_bool(ID_constructor);
+  }
+
+  void set_is_constructor(bool is_constructor)
```

Comment:
Is this ever used with an argument other than `true`?

Reply:
No - I thought it was more consistent with other setters but happy to change to a parameter-less method with the same name if you'd prefer.

Reply:
I don't have strong opinions either way, it just seemed a bit strange to have more capability than anyone ever wanted to use.

Reply:
Fixed - I originally leaned this way too as calling `set_is_constructor(false)` implies that you can retrospectively change something to be not a constructor, which is wrong.